apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/match/MultiClassNameMatch.java

```diff
@@ -57,7 +57,7 @@ public boolean isMatch(TypeDescription typeDescription) {
         return matchClassNames.contains(typeDescription.getTypeName());
     }
 
-    public static ClassMatch byMultiClassMatch(String... classNames) {
+    public static IndirectMatch byMultiClassMatch(String... classNames) {
```

Comment:
Why need to change this?

Reply:
[Change to use it here](https://github.com/apache/skywalking/pull/5519/files#diff-0c08a8cb31ed4f15d96bd82a49fa1f5bR52)

`LogicalMatchOperation#not` only accept subtype IndirectMatc.